spec/shoryuken/middleware/server/timing_spec.rb

```diff
@@ -54,8 +54,8 @@
       end
 
       expect {
-        subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise }
-      }.to raise_error
+        subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise "Error" }
+      }.to raise_error(RuntimeError, "Error")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.